test/module/shared_model/cryptography/security_signatures_test.cpp

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "module/shared_model/builders/protobuf/test_block_builder.hpp"
+#include "module/shared_model/builders/protobuf/test_signature_builder.hpp"
+#include "module/shared_model/builders/protobuf/test_transaction_builder.hpp"
+
+/**
+ * @given Two signatures with same pub key but different signed
+ * @when  Invoke operator==
+ * @then  Expect true
+ */
+TEST(SecuritySignature, SignatureOperatorEqual) {
+  auto first_signature =
+      TestSignatureBuilder()
+          .publicKey(shared_model::crypto::PublicKey("one"))
+          .signedData(shared_model::crypto::Signed("signed_one"))
+          .build();
+
+  auto second_signature =
+      TestSignatureBuilder()
+          .publicKey(shared_model::crypto::PublicKey("one"))
+          .signedData(shared_model::crypto::Signed("signed_two"))
+          .build();
+
+  ASSERT_TRUE(first_signature == second_signature);
+}
+
+/**
+ * @given Transaction with marked signature
```

Comment:
I think better to use "given" instead of "marked".
Or may be even move everything about transaction to "when" part (e. g. given transaction, when addSignature() invoked twice with ...)